ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/Ledger/Tables.hs

```diff
@@ -8,12 +8,26 @@
 {-# LANGUAGE TypeOperators            #-}
 {-# LANGUAGE UndecidableInstances     #-}
 
--- | See @'LedgerTables'@
+-- | This module defines the 'LedgerTables', a portion of the Ledger notion of a
+-- /ledger state/ (not to confuse with our
+-- 'Ouroboros.Consensus.Ledger.Basics.LedgerState') that together with it,
```

Comment:
I already confused it with `LedgerState` :smile: What is the distinction we're trying to make here? Is `LedgerState` a particular instantiation of the _ledger state_ abstract concept?

Reply:
The /ledger state/ abstract concept is concretized in the state needed by the Ledger layer. And that is the `NewEpochState` at least after Shelley.

However when we say `LedgerState` we sometimes refer to that abstract concept and sometimes to the `LedgerState` data family in consensus (which is a wrapper over the `NewEpochState`). That's why I usually write "the Ledger _ledger state_" for the abstract concept/the `NewEpochState` on Shelley, and whenever I speak about the instantiation in Consensus I hyperlink the `LedgerState` data family.